content/Contributing to open source: An overview of the django-drip-campaings project - Bruno Michetti.md

```diff
@@ -0,0 +1,217 @@
+## Contributing to open source: An overview of the django-drip-campaigns project
+
+At Rootstrap we work with many initiatives, one of them is the Open Source one. The main goal in this case is to analyze, maintain and develop open source software (OSS). In this blog I'm going to talk about a particular project named Django Drip Campaigns and how was the process of publishing it as a [pypi](https://pypi.org/) package.  
+Pypi (that stands for Python Package Index) is a repository of software for the [Python](https://www.python.org/) programming language that helps us to find and install software developed by the Python community. One of our contributions to the OSS community is to publish and maintain the named project and I want to share that experience in this blog.
+
+### The Django Drip Campaigns Project
+
+Django Drip Campaigns Project is an open source app to create drip campaigns for email using [Django’s admin](https://docs.djangoproject.com/en/3.1/ref/contrib/admin/) interface and a User queryset. This means that using this app, you are able to easily schedule and send emails to the registered users, creating querysets in a friendly interface.  
+For example, you can very quickly configure your app to send a periodic email to the users that haven't logged in in a day or more. Or you can send a custom email to the users that are registered more than a week ago, etc. You only need to define one or more querysets in the admin page.
```

Comment:
```suggestion
For example, you can very quickly configure your app to send a periodic email to the users that haven't logged in in a day or more, send a custom email to the users that registered more than a week ago, etc. To do this, you only need to define one or more querysets in the admin page.
```
I feel a `To do this` or something similar is missing in that next sentence where you explain how to do those examples you mentioned.